### PR TITLE
Readme.md updated

### LINKED ISSUE #696: Revit_Toolkit: update Readme.md
#### Description:

Following https://github.com/BHoM/admin/issues/4

### PR DESCRIPTION
**Pull request by pawelbaran**
### Issues addressed by this PR

Closes #696

### Test files

Not needed.

### Additional comments

_Getting Started for Developers_ section is a bit extended compared to https://github.com/BHoM/admin/issues/4 - I believe this is for good but would be good to get it signed off centrally @al-fisher

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/robot-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Robot_Toolkit/Robot_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=93&branchName=master)
```
Comment (FraserGreenroyd) [inline README.md:1]: References to RobotToolkit need to be changed to Revit Toolkit

File: README.md [hunk 2]
```diff
@@ -1,26 +1,34 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/robot-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Revit_Toolkit/Revit_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=99&branchName=master)
```
Comment (al-fisher) [inline README.md:1]:
```suggestion
[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/d1y1h3ywkch3fd06/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/revit-toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Revit_Toolkit/Revit_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=99&branchName=master)
```

Comment (al-fisher) [reply README.md:1]: Hey @pawelbaran - think the above should work for you.
Sorry - yes - should have highlighed this before - not easy access this links. @IsakNaslundBh @alelom FYI for when you are doing your READMEs - let me @FraserGreenroyd know to help with badges

File: README.md [hunk 3]
```diff
@@ -30,22 +38,19 @@ You will need the following to build Revit_Toolkit:
 - [BHoM_Adapter](https://github.com/BHoM/BHoM_Adapter)
 - [Socket_Toolkit](https://github.com/BHoM/Socket_Toolkit)
 
-## Debugging ##
-To switch between Revit_Toolkit for specific Revit version use Configuration Manager:
-Debug -> Revit 2018 (default)
+Revit_Toolkit needs to be built separately for each version of Revit. To switch between version-specific Revit_Toolkit configurations use Configuration Manager:
```
Comment (al-fisher) [inline README.md:41]:
```suggestion
Revit_Toolkit needs to be built separately for each version of Revit. To switch between version and specific Revit_Toolkit configurations use Configuration Manager:  
```

Tiny suggested formatting change for readability

**Comment by al-fisher** [comment]
Following updates to badges and comments above - all looks good @pawelbaran. The additional Revit_Toolkit specific material is exactly what we need and as intended for each README
Thanks

**Comment by pawelbaran** [comment]
Thank you @al-fisher, I have committed your suggestions. PR should be ready to merge now! 👍

**Comment by al-fisher** [comment]
@FraserGreenroyd assume you are happy - can you now approve to get our two approvals on Revit ?😄

**Pull request merged**